app/homepage/contribute.tsx

```diff
@@ -15,50 +14,57 @@ export const Contribute = ({
   buttonUrl: string;
 }) => {
   return (
-    <Box sx={{ backgroundColor: "primary.main", color: "grey.100" }}>
-      <Box sx={{ maxWidth: 1024, margin: "0 auto" }}>
-        <Flex sx={{ flexDirection: ["column", "row"], px: 4, py: [6, 6, 7] }}>
-          <Box sx={{ width: ["100%", "100%", "70%"], mb: [6, 6, 0] }}>
-            <Typography
-              sx={{
-                fontSize: ["1.5rem", "1.5rem", "2rem"],
-                lineHeight: 1.25,
-
-                mb: 3,
-              }}
-            >
-              {headline}
-            </Typography>
-            <Typography component="div" variant="body1">
-              {description}
-            </Typography>
-          </Box>
-          <Flex
-            sx={{
-              justifyContent: "flex-end",
-              alignItems: "center",
-              width: ["100%", "50%", "30%"],
-            }}
-          >
-            <Button
-              component={Link}
-              href={buttonUrl}
-              target="_blank"
-              variant="inverted"
-              rel="noopener noreferrer"
-              sx={{
-                flexGrow: [1, 0, 0],
-                textDecoration: "none",
-                textAlign: "center",
-                bgcolor: (theme) => theme.palette.primary.contrastText,
-                color: (theme) => theme.palette.primary.main,
-              }}
-            >
-              {buttonLabel}
-            </Button>
-          </Flex>
-        </Flex>
-      </Box>
-    </Box>
+    <Flex
+      sx={{
+        flexDirection: "column",
+        justifyContent: "space-between",
+        gap: "48px",
+        width: "100%",
+      }}
+    >
+      <Flex
+        sx={{
+          width: ["100%", "100%", "70%"],
+          flexDirection: "column",
+          gap: 4,
+          maxWidth: ["100%", "330px"],
+        }}
+      >
+        <Typography
+          sx={{
+            fontSize: ["1.5rem", "1.5rem", "2rem"],
+            lineHeight: ["2.5rem", "2.5rem", "3rem"],
+          }}
+        >
+          {headline}
+        </Typography>
+        <Typography component="div" variant="body1">
+          {description}
+        </Typography>
+      </Flex>
+      <Flex
+        sx={{
+          width: ["100%", "100%", "70%"],
+          maxWidth: ["100%", "330px"],
+          paddingBottom: ["72px", "0px"],
+          borderBottom: ["1px solid #e5e5e5", "none"],
```

Comment:
Nit: Theme palette 🎨

Reply:
Also, I wonder if it wouldn't be better to decouple border from the component and simply render a `<Divider />` between the elements? What if we need to change the order of sections? 👀